justfile

```diff
@@ -152,12 +152,66 @@ web-dev-up:
     rm "./web-api/assets" || :
     ln -s "./assets/smbert_v0003" "./web-api/assets"
     compose="$(command -v podman-compose || command -v docker-compose)"
-    $compose -f "./web-api/compose.yml" up --detach --remove-orphans
+    $compose -f "./web-api/compose.db.yml" up --detach --remove-orphans
 
 web-dev-down:
     #!/usr/bin/env -S bash -eux -o pipefail
     compose="$(command -v podman-compose || command -v docker-compose)"
-    $compose -f "./web-api/compose.yml" down
+    $compose -f "./web-api/compose.db.yml" down
+
+oci-cargo-install $CRATE_PATH $BIN $ASSET_DIR="":
+    #!/usr/bin/env -S bash -eux -o pipefail
+    ociBuilder="$(command -v podman || command -v docker)"
+    # if this is nested add more .. to the soft-link below
+    out=oci-cargo-install-build-dir
+    # cleanup from previous builds
+    [ -d "$out/bin" ] && rm -r "$out/bin"
+    # if it's a link remove it
+    [ -L "$out/assets" ] && rm "$out/assets"
+    # if it's a dummy dir remove it
+    [ -d "$out/assets" ] && rmdir "$out/assets"
+    # remove old install metadata
+    rm -f "$out/.crates"{".toml","2.json"}
```

Comment:
right, I had that initially because of how docker contexts work but I change the approach a bit so now we can have that (the temp dir still needs to be under the project root, i.e. we use dirs in `/tmp`)